cli/cmd/compliance_aws.go

```diff
@@ -39,7 +40,7 @@ var (
 	compAwsCmdState = struct {
 		Type       string
 		ReportName string
-	}{Type: "AWS_CIS_14"}
+	}{ReportName: "CIS Amazon Web Services Foundations Benchmark v1.4.0"}
```

Comment:
yikes, this is such a long name and it is being used in multiple places, instead of hardcoding it, can we use a constant either from the `api` package, if any, or declare one here in the `cli` package